### DISCUSSION TITLE
Kendala pada Signin?error=OAuthCallback

### DISCUSSION THREAD

**Discussion by AryaNurarif112** [deaafrizal/forum#66] [Troubleshooting Room]
Permisi selamat pagi menjelang siang mas mas semua, saya ada kendala di signin yang menyebabkan error OAuthCallback

<img width="1272" alt="Screen Shot 2024-02-06 at 09 56 52" src="https://github.com/deaaprizal/forum/assets/77497867/3da6ac87-5274-4a19-af54-a5ae17b62030">

**Comment by AryaNurarif112** [comment]
Dan ini untuk codingan nya

<img width="1052" alt="Screen Shot 2024-02-06 at 10 23 42" src="https://github.com/deaaprizal/forum/assets/77497867/c0c0d210-dd53-44a2-8381-79858f11f0ac">
<img width="1060" alt="Screen Shot 2024-02-06 at 10 25 09" src="https://github.com/deaaprizal/forum/assets/77497867/5b6f3c46-fd8b-4fbc-9ecc-c2190b5d43ae">

**Comment by danielpsrb** [comment]
sudah coba atur di github , url homepage dan redirect url nya ?  disesuaikan
